init.ps1

```diff
@@ -74,11 +74,19 @@ appwrite project create-variable --key APPWRITE_API_KEY --value $secret
 appwrite project create-variable --key APPWRITE_ENDPOINT --value "http://host.docker.internal:80/v1"
 
 # Pushing the project's core defined in appwrite.json
-appwrite deploy collection
-appwrite deploy bucket
-appwrite storage create-file --bucket-id "64a13095a4c87fd78bc6" --file-id "67012e19003d00f39e12" --file "pink_profile_image.jpeg"
-appwrite storage create-file --bucket-id "64a13095a4c87fd78bc6" --file-id "67012e19003d00f39e12" --file "story.png"
-appwrite storage create-file --bucket-id "64a13095a4c87fd78bc6" --file-id "67012e19003d00f39e12" --file "chapter.png"
+appwrite push collections
+appwrite push buckets
+$bucketId = "64a13095a4c87fd78bc6"
+$files = @("amber_profile_image.jpeg", "classic_profile_image.jpeg", "cream_profile_image.jpeg", "forest_profile_image.jpeg", "time_profile_image.jpeg", "vintage_profile_image.jpeg", "story.png", "chapter.png")
+
+foreach ($file in $files) {
+    # Create a unique file ID for each file
+    $fileId = "unique()"
```

Comment:
@akshatbajetha file id's should not be dynamic as if they are then the contributor would have to manually feed them in the frontend as each time the script runs the id's respective to default profile avatars will change leading the UI to crash if not updated with the new Id's

Reply:
@Aarush-Acharya so how about I just manually add different ID's to all the images being uploaded because right now in the current code, there is same ID of all the images being uploaded which leads to only one image being uploaded on the server.

![{DCBAC4BD-E8CD-4640-8CB5-A10887BA3256}](https://github.com/user-attachments/assets/6630e515-6ff6-4e75-8470-130c3044b6c2)

Reply:
@akshatbajetha  Yes this looks great 

Make sure to address the default profile avatar background concern that I rose up in the comment on the main thread

Then push a new commit and will have it merged